Tests/Reqnroll.VisualStudio.VsxStubs/StubProjectBindingRegistryCache.cs

```diff
@@ -1,28 +1,30 @@
 #nullable enable
 namespace Reqnroll.VisualStudio.VsxStubs;
 
-public class StubProjectBindingRegistryCache : Mock<IProjectBindingRegistryCache>, IProjectBindingRegistryCache
+public class StubProjectBindingRegistryCache : IProjectBindingRegistryCache
 {
-    public StubProjectBindingRegistryCache() : base(MockBehavior.Strict)
+    private readonly IProjectBindingRegistryCache _substitute;
+
+    public StubProjectBindingRegistryCache()
     {
+        _substitute = Substitute.For<IProjectBindingRegistryCache>();
+
         Value = ProjectBindingRegistry.Invalid;
-        Setup(c => c.Update(It.IsAny<Func<ProjectBindingRegistry, Task<ProjectBindingRegistry>>>()))
-            .Returns(async (Func<ProjectBindingRegistry, Task<ProjectBindingRegistry>> updateFunc) =>
-            {
-                Value = await updateFunc(Value);
-                return Task.FromResult(Value);
-            });
+        _substitute.Update(Arg.Any<Func<ProjectBindingRegistry, Task<ProjectBindingRegistry>>>())
+                   .Returns(async callInfo =>
+                   {
+                       var updateFunc = callInfo.Arg<Func<ProjectBindingRegistry, Task<ProjectBindingRegistry>>>();
+                       Value = await updateFunc(Value);
```

Comment:
This line has been changed and I'm not following why it was necessary to do so. (Could be naivete on my part) Why not follow the pattern in the original text?